ui/apps/platform/src/reducers/systemConfig.js

```diff
@@ -11,8 +12,20 @@ export const types = {
 
 // Actions
 
-export const actions = {
-    fetchPublicConfig: createFetchingActions(types.FETCH_PUBLIC_CONFIG),
+export const fetchPublicConfig = () => {
+    return async (dispatch) => {
+        dispatch({ type: types.FETCH_PUBLIC_CONFIG.REQUEST });
+
+        try {
+            const result = await fetchPublicConfigService();
+            dispatch({
+                type: types.FETCH_PUBLIC_CONFIG.SUCCESS,
+                response: result.response,
```

Comment:
Action object here and in SystemConfigForm.tsx file might become simpler with possible future change:
* from
    ```js
    export function fetchPublicConfig(): Promise<{ response: PublicConfig }> {
        return axios.get<PublicConfig>(`${baseUrl}/public`).then(({ data }) => ({
            response: data,
        }));
    }
    ```
* to
    ```js
    export function fetchPublicConfig(): Promise<PublicConfig> {
        return axios.get<PublicConfig>(`${baseUrl}/public`).then((response) => response.data);
    }
    ```